### PR TITLE
fixes for various typos in the Applied Visual Design section

### PR DESCRIPTION
**Pull request by erichartline**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of freeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [ ] Closes currently open issue (replace XXXX with an issue no): Closes #XXXX

#### Description

I fixed typos on the following pages:
Applied Visual Design: Adjust the Background-color Property of Text
Applied Visual Design: Learn about Tertiary Colors
Applied Visual Design: Adjusting the Color of Various Elements to Complementary Colors
Applied Visual Design: Adjust the Hue of a Color
Applied Visual Design: Create a Gradual CSS Linear Gradient
Applied Visual Design: Use the CSS Transform Property to Scale an Element on Hover
Applied Visual Design: Create a More Complex Shape Using CSS and HTML
Applied Visual Design: Learn How CSS Keyframes and Animation Properties Work
Applied Visual Design: Make a CSS Heartbeat using an Infinite Animation Count
Applied Visual Design: Learn How Bezier Curves Work
Applied Visual Design: Use a Bezier Curve to Move a Graphic

I also added a link to Google Fonts on the Applied Visual Design: Use a Google Font page. Feels like it would be helpful so the camper can see the site for themselves.

### CODE REVIEW

File: seed/challenges/01-responsive-web-design/applied-visual-design.json [hunk 1]
```diff
@@ -1143,7 +1143,7 @@
         }
       ],
       "description": [
-        "<code>Google Fonts</code> is a great way to find free fonts to use in a project.",
+        "<code><a href='https://fonts.google.com/' target='_blank'>Google Fonts</a></code> is a great way to find free fonts to use in a project.",
```
Comment (Greenheart) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: I'm not sure about this formatting. I tried removing the wrapping `<code>` locally, but it broke the paragraph into separate lines.

Ideally, I think this link should behave like a `display: inline` and without being wrapped in a `<code>` tag.

**Review by Greenheart — CHANGES_REQUESTED** [review]
@wildlifehexagon Nice work! :smile:

There's just one small thing that needs further discussion.

- [X] `npm test` passes
- [X] Tested locally

File: seed/challenges/01-responsive-web-design/applied-visual-design.json (hunk 1 above)
Comment (wildlifehexagon) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: Yeah that was something I was wondering about. Should I just take the link out for now?

Comment (Greenheart) [inline seed/challenges/01-responsive-web-design/applied-visual-design.json]: I think we should add it now when we're at it :blush:

@freeCodeCamp/moderators What do you think about this link? How should it be formatted?

**Comment by Bouncey** [comment]
I'm not at my computer at the minute, but does making the `code` tags a
child if the `a` tag help with formatting?

**Comment by Greenheart** [comment]
@Bouncey It does! Thanks for the quick reply :smile:

@wildlifehexagon If you follow @Bouncey's suggestions, `git commit --amend` and `git push origin <your-branch> -f`, I'd say this is ready to be merged :blush:

**Comment by wildlifehexagon** [comment]
@Bouncey @Greenheart 

Thanks for the help! I made the change, but I have a question before I re-submit. I noticed that there is a square box after the link now -- is that supposed to be there?

Here's what I'm seeing: http://imgur.com/a/a37ln

**Comment by Greenheart** [comment]
It's a link icon (square with arrow going from center to top right), which
I think I've seen for external links before.

It just seems like it didn't load properly in your browser. Might be a
network- or browser extension-issue.

lördag 28 januari 2017 skrev Eric Hartline <notifications@github.com>:

**Comment by Greenheart** [comment]
@wildlifehexagon I just realized, we might not even need the `<code>` at all. Without it, the `<a>` gets it's normal blue color which makes sense.

What would you say about removing `<code>`? :smile:

**Comment by wildlifehexagon** [comment]
@Greenheart Great point, the code isn't really necessary. I'll resend without it.

**Review by Greenheart — APPROVED** [review]

**Comment by Greenheart** [comment]
@wildlifehexagon But now we know it works at least :wink: 

Thanks for fixing this!

**Pull request merged**